lib/internal/errors.js

```diff
@@ -47,7 +47,8 @@ function lazyBuffer() {
 // and may have .path and .dest.
 class SystemError extends Error {
   constructor(key, context) {
-    const prefix = getMessage(key, []);
+    super();
+    const prefix = getMessage.call(this, key, []);
```

Comment:
Can we remove the `call()`?

Reply:
Sure, I could use `getMessage(self, key, [])` instead?

Reply:
Updated